source/frontend/package.json

```diff
@@ -126,7 +126,7 @@
     "start": "react-app-rewired start",
     "build": "cross-env CI=true GENERATE_SOURCEMAP=false INLINE_RUNTIME_CHUNK=false react-app-rewired build",
     "test": "tsc && react-app-rewired test --ci --watchAll=false",
-    "test-u": "tsc && react-app-rewired test --ci --watchAll=false -u --detectOpenHandles",
+    "test-u": "tsc && react-app-rewired test --ci --watchAll=false -u",
```

Comment:
?

Reply:
Added this by accident last time. Fixing it now.